File: 2018-12-21/4344/4344.c
#include <stdio.h>

int main(){
	int N, C ;
	float avg, score[1001], sum, largeNum;
	scanf("%d", &C);
	for (int test = 0; test<C; test++){
		scanf("%d", &N);
		sum = 0;
		largeNum = 0;
		for(int person = 0; person < N;person++){
			scanf("%f", &score[person]);
			sum += score[person];
		}
		avg = sum/N;
		for(int person = 0; person < N;person++){
			if(score[person] > avg)
				largeNum +=1;
		}
		printf("%.3f%%\n", (largeNum/N)*100);
	}
}

File: 2018-12-21/4344/4344.py
import sys

N = int(sys.stdin.readline())

for i in range(0, N):
	score = list(map(int, sys.stdin.readline().split()))
	sumScore = sum(score[1:])
	avg = sumScore/score[0]
	filterScore = [i for i in score[1:] if i>avg]
	print("%.3f%%"%(len(filterScore)/score[0]*100))

	